### PR TITLE
Add observation transformations and update to db

### LINKED ISSUE #51: Migrate persisting update step
More or less copy / paste from ert-storage

### PR DESCRIPTION
**Pull request by ManInFez**
resolves #51

### CODE REVIEW

File: ert_storage/endpoints/ensembles.py [hunk 1]
```diff
@@ -14,5 +13,63 @@ def post_ensemble(
...
+        children=[child.ensemble_result_id for child in ens.children],
+        parent=ens.parent.ensemble_reference_id if ens.parent else None,
+    )
+
+
+@router.post("/ensembles/{id}/updates", response_model=js.UpdateOut)
+def create_update(
+    *,
+    db: Session = Depends(get_db),
+    id: int,
```
Comment (oysteoh) [inline ert_storage/endpoints/ensembles.py:44]: Is this one necessary? For me, it seems like this id should contain the ensemble to attach the update to. But from what i can see on line 50 you are getting this from the `UpdateIn` object through `ensemble_reference_id`. Do I misunderstand something? 🤔

Comment (ManInFez) [reply ert_storage/endpoints/ensembles.py:44]: yes you are right

File: ert_storage/json_schema/observation.py [hunk 2]
```diff
@@ -3,6 +3,24 @@
 from .record import RecordOut
```
Comment (oysteoh) [inline ert_storage/json_schema/observation.py:3]: Not your change, but this import is never used

Comment (ManInFez) [reply ert_storage/json_schema/observation.py:3]: I can include it

**Review by oysteoh — APPROVED** [review]
LGTM 🚀

**Pull request merged**